content/efcore/getting-started/samples/manipulating-entities/adding-related-entities/Program.cs

```diff
@@ -28,9 +29,11 @@ public static void Main()
             var addedAuthor = context.Authors
                 .Include(a => a.Books)
                 .Single(a => a.LastName.Contains("Shelley"));
+				
+			Console.WriteLine("We added the author {0} {1} with their book(s) -", addedAuthor.FirstName, addedAuthor.LastName);
```

Comment:
For our output design, we use dashes for the initial label or phrase, then colons for any following labels or if there is only a single one.